nflows/transforms/splines/rational_quadratic.py

```diff
@@ -57,12 +58,29 @@ def unconstrained_rational_quadratic_spline(
             min_bin_width=min_bin_width,
             min_bin_height=min_bin_height,
             min_derivative=min_derivative,
-            enable_identity_init=enable_identity_init,
         )
+        
+        
+        # turn inside_interval_mask into an int tensor
```

Comment:
Could we simplify below by using [masked_scatter](https://pytorch.org/docs/stable/generated/torch.Tensor.masked_scatter.html)?